pyvista/_vtk.py

```diff
@@ -548,6 +558,14 @@ def lazy_vtkCGNSReader():
         """Lazy import of the vtkCGNSReader."""
         raise VTKVersionError('vtk.CGNSReader requires VTK v9.1.0 or newer')
 
+    def lazy_vtkPOpenFOAMReader():
```

Comment:
The current structure of this module is going to be increasingly hard to maintain with different vtk versions.  This shouldn't be fixed in this PR, but I'm wondering whether it is better to keep the current PR proposal which will be required when vtk 9.2 is released, with additional changes here, to take advantage of the upstream fix.  Or, to simplify this PR too have one function outside the `if` `else` block here.  Thoughts?

Reply:
We will probably need something like this when 9.2.0 is released, but I don't want to overcomplicate this PR now.  I think we should have one definition of `lazy_vtkPOpenFOAMReader`, since the implementation for vtk version >= 9 is the same as for version < 9.  This can change in version 9.2, but I think we should refactor this entire module at that point.

Reply:
Oh I misread the implementation sorry.  This is good as is